[docs] Update Get started tutorial (#31382)
expo/expo: docs/pages/tutorial/follow-up.mdx

import { A } from '~/ui/components/Text';

Now that the example app is done, let's learn more about the technologies we used to build it.

## Build your project into an app

To start creating a new app, you can either:

> - [Sign-up to expo.new](https://expo.new/) which automatically configures a project with file-based routing, and the required setup for build and over-the-air updates.

Author: Done.

But for OTA, see this: https://docs.expo.dev/deploy/send-over-the-air-updates/. We've started using it again.

- Or, start locally on your machine and use `npx create-expo-app@latest` and [set up your development environment](/get-started/set-up-your-environment/) sequentially.

### Recommended resources

Once you've created your new project, you need to learn more about the concepts following tools and concepts:

- [Development tools](/develop/tools/): A reference of Expo tools that will help you during various aspects of your app-building journey.
- [Development builds](/develop/development-builds/introduction/): Using a development build allows you to gain full control over your app's build process, and to test your app on a device or simulator.

> - [Key concepts](/workflow/overview/#key-concepts) and [Core development loop](/workflow/overview/#the-core-development-loop): If you want to build a mental model of how Expo app is created, go through key concepts and core development loop to understand how it works.

Author: My idea behind this was people, after getting back knowledge, start questioning (mob thinking) such as how do I add library X or how do I add third party library, will it work Expo, where are android and ios directories in my project, do I need them, etc., was to address these kind of questions by surfacing the info from development overview. Also, core development loop described on that page supersedes the development loop this tutorial goes through. 

Maybe I should just link to Development overview page and mention the key concepts 🤔

> - [Key concepts](/workflow/overview/#key-concepts) and [Core development loop](/workflow/overview/#the-core-development-loop): If you want to build a mental model of how Expo app is created, go through key concepts and core development loop to understand how it works.

Author: I've changed this to "development overview" page and adjusted the description accordingly. I think, it is important for us to mention key concepts and core development loop because, even though the development loop of an app may vary, it is still important for folks to know where this information lives inside our docs and what they can be expect from the core development loop (or at least how we explain this information) in terms of Expo.

- [Expo Router](/router/introduction/): We went through basics of Expo Router and implemented a tab navigator. See its documentation to learn more about the library.
- [App icon](/develop/user-interface/splash-screen-and-app-icon/#app-icon) and [splash screen](/develop/user-interface/splash-screen-and-app-icon/#splash-screen): You can learn more about customizing your app icon and splash screen guides. Also, look through the [app config reference](/workflow/configuration) for properties you can configure in the **app.json** file.

> - [App distribution](/deploy/build-project/) and [submission](/deploy/submit-to-app-stores/) to app stores: Once you've built your project, you are ready to share it with your users. Read these resources to learn more about how to release and submit your app to app stores.

Reviewer: ```suggestion
- [App distribution](/deploy/build-project/) and [submission](/deploy/submit-to-app-stores/) to app stores: Read these resources to learn more about how to release and submit your app to app stores once it's ready to ship.
```
